src/main/resources/application.yaml

```diff
@@ -20,6 +20,7 @@ spring:
           cache:
             missing_cache_strategy: create
     hibernate.ddl-auto: validate
+    open-in-view: false
```

Comment:
It's actually good, to set it to false...
But now you have to perform manual regression testing of the application to make sure that we don't rely on this feature anywhere.

Reply:
Yes, I tested the app manually, I didn't find bugs so far